crates/viewer/re_ui/src/markdown_utils.rs

```diff
@@ -0,0 +1,47 @@
+use std::fmt::{Display, Formatter};
+
+/// Newtype over [`egui::PointerButton`] which provides a [`Display`] implementation suitable for
+/// markdown.
+pub struct MouseButtonMarkdown(pub egui::PointerButton);
```

Comment:
Using `Display` for markdown seems a bad path to go down.

There are many risk for subtle bugs when we put markdown in plain strings. We should either consider using a newtype (`struct Markdown(pub String)`) or we need to be principled and be very explicit whenever we return markdown from a function.

In either case, I'd prefer calling `.to_markdown()` to get markdown, rather than `.to_string()`

Reply:
Do you have a concrete proposal for that?

In principle I agree, but in practice I'm unsure what exact steps to take here. One of my goal is to make  `help_markdown()` as easy to implement/maintain, such as it no longer rots like it has so far. Implementing `Display` is part of that, it reduces the boilerplate in the `format!` (which is already greater than I'd wish). Also, we are already pretty explicit, with the types implementing `Display` having `Markdown` in full in their name.

Reply:
Ah, I didn't really notice how this was used, and now I agree it is pretty good (with the explicit naming). Nevermind me :)